### DISCUSSION TITLE
Rebuild artifacts for CI Buil?

### DISCUSSION THREAD

**Discussion by gwolanin** [forkineye/ESPixelStick#791] [General]
Is there a way to rebuild the artifacts for the CI build?  They look expired.

**Comment by MartinMueller2003** [comment]
You would need to set up a build on your own PC. Then you can grab the source files based on a commit or date and build an image for your platform.

**Comment by forkineye** [comment]
I just kicked off a manual build, they should be there soon.

**Reply by gwolanin** [reply]
Thank you!
